src/Raven.Server/Raven.Server.csproj

```diff
@@ -136,6 +136,11 @@
     <PackageReference Include="AWSSDK.Glacier" Version="3.7.0.148" />
     <PackageReference Include="AWSSDK.S3" Version="3.7.8.19" />
     <PackageReference Include="Azure.Storage.Blobs" Version="12.11.0" />
+    <PackageReference Include="AWSSDK.Glacier" Version="3.7.0.110" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.7.5" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.10.0" />
```

Comment:
Was this intentional?

Reply:
Note two lines above, we have this with different versions

Reply:
Please check it again. It looks we're still having duplicated entries here. Please make sure we add only the relevant packages